BE-workspace/src/main/resources/schema.sql

```diff
@@ -0,0 +1,23 @@
+drop table if exists user;
+drop table if exists interest;
+
+create table user (
+    id bigint auto_increment primary key,
+    user_iD varchar(64),
```

Comment:
```suggestion
    user_id varchar(64) UNIQUE,
```

아까 위 코드를 잠시 보니 user_id, email, mobile은 unique여야 할 것 같아요